Can't resolve duckdb

I had a workflow with esbuild bundling  hpcc/wasm and the d3-graphviz and some other code. 

Since updating the packages I'm getting an error when running the bundler: 
```
✘ [ERROR] Could not resolve "./duckdb.js"

    node_modules/@hpcc-js/wasm/dist/index.js:3:893872:
      3 │ ...s=>s.Base91.load())}i.load=o})(Sa||={});var Qa;(i=>{function o(){return import("./duckdb.js").then(s=>s.DuckDB.load())}i.load=o})(Qa||={});var Ja;(i=>{function o(){return ...
        ╵                                                                                   ~~~~~~~~~~~~~ 
        ```
        It looks like it's trying to find a local duckdb.js in the dist folder but can't resolve it? The other imports base91.js and expat.js all seem to be present.

Cheers

--- a/packages/wasm/esbuild.mjs
+++ b/packages/wasm/esbuild.mjs
@@ -3,6 +3,7 @@ import { bothTpl, browserBoth, nodeBoth } from "@hpcc-js/esbuild-plugins";
 //  config  ---
 await Promise.all([
     bothTpl("src/base91.ts", "dist/base91", undefined, "@hpcc-js/wasm"),
+    bothTpl("src/duckdb.ts", "dist/duckdb", undefined, "@hpcc-js/wasm"),
     bothTpl("src/graphviz.ts", "dist/graphviz", undefined, "@hpcc-js/wasm"),
     bothTpl("src/expat.ts", "dist/expat", undefined, "@hpcc-js/wasm"),
     bothTpl("src/zstd.ts", "dist/zstd", undefined, "@hpcc-js/wasm")

